packages/pluggableWidgets/datagrid-web/src/utils/__tests__/settings.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { useSettings } from "../settings";
+import { ColumnWidth, TableColumn } from "../../components/Table";
+import { EditableValueBuilder } from "@native-mobile-resources/util-widgets";
+import { HidableEnum } from "../../../typings/DatagridProps";
+import { renderHook } from "@testing-library/react-hooks";
+
+describe("useSettings Hook", () => {
+    it("should load correct values into hooks", () => {
```

Comment:
🎄 `it("loads correct values into hooks")` (same for other it's)

Reply:
Fixed